package.json

```diff
@@ -34,6 +34,9 @@
   "pnpm": {
     "overrides": {
       "typescript-json-schema>@types/node": "*"
+    },
+    "patchedDependencies": {
```

Comment:
Thanks for adding this in, should we add some comments about this patch as well?

I'm not very familiar with patches in this manner, the `.patch` file was written by you manually, is that right? Or was it recommended somewhere?

Reply:
It was a known problem with mui and three-fiber, the only solution I found is to do a hacky patch.